.env.local.example

```
@@ -0,0 +1,15 @@
+# RPC urls, follow the format RPC_URL_{chainId}={rpcUrl}
+
+PONDER_RPC_URL_1=https://ethereum-rpc.publicnode.com
+PONDER_RPC_URL_8453=https://base-rpc.publicnode.com
+PONDER_RPC_URL_59144=https://linea-rpc.publicnode.com
+
+# ponder indexer ens deployment configuration
+
+INDEX_BASENAME=.base.eth
```

```suggestion
INDEX_SUBNAMES=base.eth
```

Two suggestions here:

1. Rename `INDEX_BASENAME` to `INDEX_SUBNAMES`: "subnames" is more common language in ENS and "base" already has multiple potential interpretations in our project.
2. Don't prefix the value with "." In other words, make it "base.eth" instead of ".base.eth". This way the value is a valid ENS name.